data/card_catalogue.json

```diff
@@ -413,33 +475,23 @@
         {
           "title": "Ender Pearl",
           "upgrade": "",
-          "tier": "grey",
+          "tier": "blue",
           "details": {
-            "rules": "Teleport to a space up to 10 tiles away and take 2 damage",
+            "rules": "Teleport to any visible space. Take damage equal to the number of squares moved.",
             "flavour": "Travelling is always a pain",
             "image": "assets/core/card/illustration/Ender Pearl.png"
           }
         },
         {
           "title": "Poltergeist",
           "upgrade": "",
-          "tier": "grey",
+          "tier": "green",
           "details": {
-            "rules": "Possess a non-magic object within 5 tiles of your body until the end of your turn",
+            "rules": "Possess a non-magic object within 5 tiles of your body until you lose concentration.",
             "flavour": "\"There's something rather.. lively about this chair\" - General Kudzo",
             "image": "assets/core/card/illustration/Poltergeist.png"
           }
         },
-        {
-          "title": "Beater Barter",
-          "upgrade": "",
-          "tier": "green",
-          "details": {
-            "rules": "The next time you're unconscious this combat, revive all other unconscious players. They get their lost cards returned to them.",
-            "flavour": "And on that day he realized that what his people needed wasn't a hero, but a martyr.",
-            "image": "assets/core/card/illustration/Beater Barter.png"
-          }
-        },
```

Comment:
Cool idea, but too niche IMO. I don't forsee a scenario where two players die at the same time ever. It's also not super intuitive